frontend/src/Page.js

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import logo from './uj_logo.png';
```

Comment:
Ezt nem látom a PR-ben korábban ment be vagy lemaradt?

Reply:
ezt az uj_logo.png-t én tettem fel korábban ;)

Reply:
No action needed